books/free-programming-books-langs.md

```diff
@@ -2444,6 +2445,11 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [UNIX Tutorial for Beginners](http://www.ee.surrey.ac.uk/Teaching/Unix/)
 
 
+### V
+
+* [V Documentation](https://github.com/vlang/v/blob/master/doc/docs.md) (Markdown)
```

Comment:
For repositories better use HEAD ref, because it's more mantenible.

```suggestion
* [V Documentation](https://github.com/vlang/v/blob/HEAD/doc/docs.md) - Alex Medveniko, vlang.io, et al. (Markdown)
```
Added authoring too